Make.inc

```diff
@@ -182,8 +182,46 @@ endif
 # Whether to use GPL libraries or not.
 USE_GPL_LIBS ?= 1
 
+# Whether to install Julia as a framework on Darwin (Apple) platforms.
+DARWIN_FRAMEWORK ?= 0
+
+# Override in Make.user to customize the framework:
+FRAMEWORK_NAME ?= Julia
+ifeq ($(JULIA_MAJOR_VERSION),0)
+FRAMEWORK_VERSION ?= $(JULIA_MAJOR_VERSION).$(JULIA_MINOR_VERSION).$(JULIA_PATCH_VERSION)
+else
+FRAMEWORK_VERSION ?= $(JULIA_MAJOR_VERSION).$(JULIA_MINOR_VERSION)
+endif
+
+# X.Y.Z or X.Y.Z-hash
+JULIA_VERSION_OPT_COMMIT:=$(shell [ $$(git describe --tags --exact-match 2>/dev/null) ] && echo $(JULIA_VERSION) || echo $(JULIA_VERSION)-$(JULIA_COMMIT))
```

Comment:
I'm thinking it's probably cleaner just to use `$(JULIA_COMMIT)` here; we already mimick a bunch of this logic immediately above, and this line won't play well with `make NO_GIT=1`.